styles.css

```diff
@@ -336,7 +336,13 @@
 
 .path-mod-groupformation #accordion li a {
     float: left;
-    padding: 10px 20px 0 0 ;

```

Comment:
Thanks for the great contribution(s). For my testing: which Moodle theme are you using? (I asume your theme caused the made improvements to be necessary)